src/js/revert.js

```diff
@@ -0,0 +1,38 @@
+/*
+add listener to the power button and send request to tab based on power on or power off event
+*/
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Instead of adding the listener here, I suggest this file just contains the general function that does the logic when you click the power button which takes the tab id as parameter, then in _popup.js_ you can associate the function to the image element using jQuery. That way you can use the tab id we have already in there and don't have to repeatedly call the asynchronous functions from `chrome.tabs`